ci/release/_install.sh

```diff
@@ -82,6 +82,7 @@ Flags:
   as for installation. i.e if you used --method standalone you must again use --method
   standalone for uninstallation. With detect, the install script will try to use the OS
   package manager to uninstall instead.
+  note: tala will also be uninstalled if installed.
```

Comment:
actually, can we change this to: "all binary plugins will also be uninstalled, if any"

Reply:
No, there's no concept of binary plugins in the install script. There's just d2 and tala. We do not need to complicate the docs until necessary.